src/Form/grid/StyledContainer/index.tsx

```diff
@@ -52,6 +62,84 @@ export const StyledContainer = forwardRef<HTMLDivElement, StyledContainerProps>(
 
     useContainerEngine(node, rawNode, ref);
 
+    useEffect(() => {
+      nodeRef.current = node || rawNode;
+    }, [node, rawNode]);
+
+    useEffect(() => {
+      let _node = getNode();
+
+      if (_node.uuid || !isFixed) {
+        return; // Disable all fixed behaviour on the editor
+      }
+
+      if (fixedContainerRef.current) {
+        const doc = featheryDoc();
+        const container = doc.querySelector(`div[data-id="${_node.id}"]`);
+
+        if (container) {
+          // If the container is the first child of root, top/left will be 0
+          // If the container is the last child of root, bottom/left will be 0
+          // If the container is none of the above, top/left will be taken from the container
+          // Width will be taken from the container
+
+          const setPositionAndDimensions = () => {
+            if (!fixedContainerRef.current || !container) {
+              return;
+            }
+
+            _node = getNode();
+
+            const { top, left } = container.getBoundingClientRect();
+            const isParentRoot = !_node.parent.parent;
+            const isFirstChild = _node.parent.children[0].id === _node.id;
+            const isLastChild =
+              _node.parent.children[_node.parent.children.length - 1].id ===
+              _node.id;
+
+            if (isParentRoot && isFirstChild) {
+              fixedContainerRef.current.style.top = '0';
+              fixedContainerRef.current.style.left = `${left}px`;
+            }
+
+            if (isParentRoot && isLastChild) {
```

Comment:
should probably be else ifs. also would be great to add a comment explaining the snapping logic